chunk_reader_test.go

```diff
@@ -109,7 +109,7 @@ func exampleEncode(plaintext []byte) []byte {
 	return buf.Bytes()
 }
 
-func ExampleStream() {
+func Example_stream() {
```

Comment:
go vet expects `ExampleThing` to refer to an existing function/variable `Thing`.